src/test/java/studentlist/StudentListTest.java

```diff
@@ -90,4 +92,80 @@ public void testMakeStudent() {
         assertEquals(4.5f, student.getGpa());
         assertEquals(3, student.getUniPreferences().size());
     }
+
+    @Test
+    public void testMakeStudent_idWithSpaces() {
```

Comment:
It seems like you are following `unitBeingTested_descriptionOfTestInputs_expectedOutcome` format from other test cases. This one seems to be missing expectedOutcome.
```suggestion
    public void makeStudent_idWithSpaces_studentCreated() {
```